src/components/Rewards/Rewards/Rewards.js

```diff
@@ -0,0 +1,89 @@
+import CurveA from "../../../assets/2-shapes/curve-a.svg";
```

Comment:
Link to this page in the Sidebar
![image](https://user-images.githubusercontent.com/66650721/184475885-8c116472-86ed-48c1-b14d-2de3dfd3c665.png)

Reply:
In my PR, I'm making it so that everyone can see our Partners - let me know if we want to do this differently